impl/src/main/java/jakarta/faces/annotation/FacesViewId.java

```diff
@@ -0,0 +1,60 @@
+package jakarta.faces.annotation;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import jakarta.enterprise.util.AnnotationLiteral;
+import jakarta.inject.Qualifier;
+
+/**
+
+ *
```

Comment:
Javadoc missing?

Reply:
Indeed, I didn't add Javadoc or spec text yet. I'll do that after feedback.

Reply:
Done